index.html

```diff
@@ -38,14 +38,23 @@
   </head>
   <body>
     <section  id="abstract">
-      <p> This document describes architecture on Web of Things which derived from use cases.<br />
-      The architecture consists of building blocks and the function of each block is explained.<br />
-      And it is also described how to map the WoT architecture to the wide variety of real implementation patterns.<br />
-      <br />
-      This document focused on the standardization scope of Web of Things.<br />
-      And other blocks and conditions underlying the Web of Things are described as example implementation and/or requirements such as the description of how to treat constraint devices and security and privacy.<br />
-      Web of Things is distributed platform over the latest Web technology. And Web of Things can wrap the difference among legacy IoT protocols defined by various standardizing organizations and also can provide interoperability between above legacy standards.<br />
-      Utilizing this interoperability, IoT application developers can ship only one wot-compliant application on behalf of shipping each legacy standard-compliant application.</p>
+      <p> This document describes an architecture for the Web of Things.
+      This architecture is derived from a set of use cases and consists of
+      a set of building blocks.
+      Each building block in the architecture is defined and its function explained.
+      The WoT architecture can be mapped onto a variety of real implementation patterns, and this is also described.
+      </p><p>
+      This document is focused on the standardization scope of Web of Things,
+      which consists of three primary building blocks: 
+      Thing Descriptions, a Scripting API, and Binding Templates.
+      Other non-normative architectural blocks and conditions underlying the Web of Things are described as example implementations and/or requirements, such as a description of how to treat constrained devices.
+Non-normative recommendations for security and privacy are also outlined.
+      The Web of Things is a distributed platform using the latest Web technology.
+      The Web of Things also provides for interoperation with other IoT protocols.
+      Primarily, the Web of Things is descriptive, not prescriptive.
+      Utilizing this interoperability, IoT application developers can ship only one wot-compliant application instead of shipping code for multiple standards.
+Conversely, by providing a Thing Description for a legacy device, it is possible to access it from a Web of Things application.

```

Comment:
Maybe we should start with the "description over prescription" philosophy (i.e., move it to the beginning).

"legacy device" is later used for devices that cannot have a TD or integrators do not want to ahsre, so they are connected through the "proprietary/legacy communication".

We used "existing device" for those which can be wrapped by a TD to become part of WoT.